packages/server/main.js

```diff
@@ -6,9 +6,62 @@ import { format, parse } from "json-rpc-protocol";
 // see https://koajs.com/
 const app = new Koa();
 
+var entries = new Map();
+let idCounter = 0;
+
+function generateId() {
+  idCounter += 1;
+}
+
 const METHODS = {
-  subtract([a, b]) {
-    return a - b;
+  createEntry([nameEntry, contentEntry]) {
+    const entry = {
+      id: idCounter,
+      name: nameEntry,
+      content: contentEntry,
+      created: Date.now(),
+      updated: Date.now(),
+    };
+
+    generateId();
+    entries.set(entry.id, entry);
+
+    return entry.id;
+  },
+
+  listEntries() {
+    return Array.from(entries);
+  },
+
+  deleteEntry([idEntry]) {
+    const entry = entries.get(+idEntry);
```

Comment:
You don't need to do a `get()` to throw an error if the entry does not exist, take a look at [`Map#delete()`](https://devdocs.io/javascript/global_objects/map/delete).